configure

```diff
@@ -480,11 +481,14 @@ def check_compiler(o):
     o['variables']['gas_version'] = get_gas_version(CC)
 
 
-def cc_macros():
-  """Checks predefined macros using the CC command."""
+def cc_macros(cc=None):
+  """Checks predefined macros using the C compiler command."""
+
+  if cc is None:
+    cc = CC
 
   try:
-    p = subprocess.Popen(shlex.split(CC) + ['-dM', '-E', '-'],
+    p = subprocess.Popen(shlex.split(cc) + ['-dM', '-E', '-'],
```

Comment:
Not really a nit, just a suggestion, but you could shorten this to `cc or CC` and drop the `if cc is None` check above.